.gitignore

```diff
@@ -1,4 +1,3 @@
 vendor
 launch.json
-echo
-enrober
```

Comment:
Why'd you remove `enrober`?  I don't see why we'd ever want to version binaries in Git.